models/migrations/v78.go

```diff
@@ -48,6 +48,9 @@ func renameRepoIsBareToIsEmpty(x *xorm.Engine) error {
 
 		if len(indexes) >= 1 {
 			_, err = sess.Exec("DROP INDEX IDX_repository_is_bare ON repository")
+			if err != nil {
```

Comment:
The `err` has been resolved on line 59

Reply:
No it is not.
A new **local** `err` var is created on line 44 in the block.

Reply:
@lunny What @ngourdon said.